### ISSUE TITLE
TDE Pause/Resume and Compat Level.

### ISSUE THREAD

**Issue by jdanton** [MicrosoftDocs/sql-docs#7134]
Would it be possible to add that compatibility level 150 is *not* requried for this feature. Question just came up on SQLHelp on Twitter. and as I far as I can test it's not a requirement. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 20953c9f-c769-7849-420a-dac77f917d4a
* Version Independent ID: dfd9ad27-844c-c3c8-58a3-5d4a0c92fbba
* Content: [Transparent Data Encryption (TDE) - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/transparent-data-encryption?view=sql-server-ver15)
* Content Source: [docs/relational-databases/security/encryption/transparent-data-encryption.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/security/encryption/transparent-data-encryption.md)
* Product: **sql**
* Technology: **security**
* GitHub Login: @shohamMSFT
* Microsoft Alias: **shohamd**

**Comment by WilliamAntonRohm** [comment]
@jdanton -- thank you for your research.

@shohamMSFT -- please look into this issue.

**Comment by shohamMSFT** [comment]
@jdanton  You are correct, any of the supported compatibility levels for SQL Server 2019 can be used with TDE Suspend/Resume. Thanks for the suggestion and we will look into whether this can be added to the document.

**Comment by shohamMSFT** [comment]
#please-close
